src/app/lib/utilities/getRecommendationsUrl/index.js

```diff
@@ -0,0 +1,4 @@
+export default ({ assetUri, variant }) =>
+  variant
+    ? `${assetUri}/recommendations/${variant}`
```

Comment:
The recommendations endpoint in Mozart doesn't yet support variants, but its nice to see it here